data/locale.yml

```
@@ -262,6 +262,10 @@ clipboard:
   copy_to_clipboard: Copy to Clipboard
   copied: Copied
 
+launch_button:
```

For more accessibility why, and where this used.

```suggestion
fiddle_launch_button:
```